<ui version="4.0" >
 <author></author>
 <comment>*********************************************************************
**
** Copyright (C) 1992-2008 Trolltech ASA. All rights reserved.
**
** This file is part of the Qt Linguist of the Qt Toolkit.
**
** This file may be used under the terms of the GNU General Public
** License versions 2.0 or 3.0 as published by the Free Software
** Foundation and appearing in the files LICENSE.GPL2 and LICENSE.GPL3
** included in the packaging of this file.  Alternatively you may (at
** your option) use any later version of the GNU General Public
** License if such license has been publicly approved by Trolltech ASA
** (or its successors, if any) and the KDE Free Qt Foundation. In
** addition, as a special exception, Trolltech gives you certain
** additional rights. These rights are described in the Trolltech GPL
** Exception version 1.2, which can be found at
** http://www.trolltech.com/products/qt/gplexception/ and in the file
** GPL_EXCEPTION.txt in this package.
**
** Please review the following information to ensure GNU General
** Public Licensing requirements will be met:
** http://trolltech.com/products/qt/licenses/licensing/opensource/. If
** you are unsure which license is appropriate for your use, please
** review the following information:
** http://trolltech.com/products/qt/licenses/licensing/licensingoverview
** or contact the sales department at sales@trolltech.com.
**
** In addition, as a special exception, Trolltech, as the sole
** copyright holder for Qt Designer, grants users of the Qt/Eclipse
** Integration plug-in the right for the Qt/Eclipse Integration to
** link to functionality provided by Qt Designer and its related
** libraries.
**
** This file is provided "AS IS" with NO WARRANTY OF ANY KIND,
** INCLUDING THE WARRANTIES OF DESIGN, MERCHANTABILITY AND FITNESS FOR
** A PARTICULAR PURPOSE. Trolltech reserves all rights not expressly
** granted herein.
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
 <exportmacro></exportmacro>
 <class>TrPreviewToolClass</class>
 <widget class="QMainWindow" name="TrPreviewToolClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qt Translation Preview Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget" />
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuViewViews" >
     <property name="title" >
      <string>&amp;Views</string>
     </property>
    </widget>
    <addaction name="menuViewViews" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAbout_Qt" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenForm" />
    <addaction name="actionLoadTranslation" />
    <addaction name="actionReloadTranslations" />
    <addaction name="separator" />
    <addaction name="actionClose" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuView" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionOpenForm" />
   <addaction name="actionLoadTranslation" />
   <addaction name="actionReloadTranslations" />
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <widget class="QDockWidget" name="dwForms" >
   <property name="windowTitle" >
    <string>Forms</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QListView" name="viewForms" >
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="uniformItemSizes" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpenForm" >
   <property name="icon" >
    <iconset resource="qttrpreviewtool.qrc" >:/images/open_form.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open Form...</string>
   </property>
  </action>
  <action name="actionLoadTranslation" >
   <property name="icon" >
    <iconset resource="qttrpreviewtool.qrc" >:/images/load_translation.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Load Translation...</string>
   </property>
  </action>
  <action name="actionReloadTranslations" >
   <property name="icon" >
    <iconset resource="qttrpreviewtool.qrc" >:/images/reload_translations.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Reload Translations</string>
   </property>
   <property name="shortcut" >
    <string>F5</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="text" >
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
